docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -32,18 +31,28 @@ TIP: To sort the rules list, click any column header. To sort in descending orde
 
 For detailed information on a rule, the alerts it generated, and associated errors, click on its name in the table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Installed Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
 
+For information about rule execution gaps (which are periods of time when a rule didn't run as expected), use the panel above the table. The time filter on the left allows you to select a time range for viewing gap data. The **Total rules with gaps:** field tells you how many rules have unfilled or partially filled gaps within the selected time range. The **Only rules with gaps** filter on the right lets you only display rules with unfilled or partially filled gaps. 
```

Comment:
> unfilled or partially filled

Maybe we can just only mention `unfilled` as it simpler to understand?

Reply:
I had considered this, but I think it's important to make the distinction that partially filled gaps are represented here too. If the docs only mentioned unfilled gaps, it could be interpreted gaps that are entirely unfilled (i.e., no partially gaps).